libs/core/tsconfig.json

```diff
@@ -2,11 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./dist",
-    "rootDir": "src",
```

Comment:
With this change, the `package.json` (placed in the root folder outside `src`) can be included and `tsc` will copy it to`dist` when building. This is much like how ng-packagr does for our angular packages, which means we can now reuse our package alias strategy and point to the core dist folder during local development. This streamlines the wait-on logic for parallel building too.